Core/HLE/sceKernelTime.cpp

```diff
@@ -171,3 +171,7 @@ u32 sceKernelLibcGettimeofday(u32 timeAddr, u32 tzAddr)
 	hleReSchedule("libc timeofday");
 	return 0;
 }
+
+u32 returnEmulatedTime() {
```

Comment:
Should be something like `KernelTimeNow` rather than `returnEmulatedTime`.  We used to use `__KernelTime` but that technically runs afoul of #1056, so we should probably skip the `__`.